### PR TITLE
Feat/dyn tier thresholds

### PR DESCRIPTION
**Pull request by ipapandinas**
This [PR](https://github.com/AstarNetwork/Astar/pull/1306) will be merged soon, I adjusted documentation accordingly. For additional context, a [forum post](https://forum.astar.network/t/dapps-staking-v3-proposal/4206/170) explains the motivation.

### CODE REVIEW

**Review by Gunit2481 — APPROVED** [review]
LGTM

File: docs/learn/dapp-staking/dapp-staking-protocol.md [hunk 1]
```diff
@@ -282,16 +282,20 @@ The comparison is always done between the _baseline_ number of slots and the new
...
 
-* `Dynamic` - adjusts the threshold based on the aforementioned formula.
-* `Fixed` - defines a static, fixed threshold which doesn't adapt.
+* `Dynamic` - A percentage of the total issuance as staked funds that can change between periods. It includes a minimum percentage that the threshold cannot fall below. This type is used for _higher_ tiers.
+* `Fixed` - A constant percentage of the total issuance as staked funds, which does not change between periods. Used for the _lowest_ tier, and defines a static value.
 
-The `Dynamic` threshold is used for _higher_ tiers, and also defines the minimum amount to which the threshold can fall.
-E.g. a threshold might define _current_ value as **1,000,000 ASTR**. Since this can be decreased if number of slots goes up,
-it also defines a minimum amount the threshold can take, e.g. **500,000 ASTR**.
+These percentages are calculated based on a total issuance of **8.4 billion ASTR** tokens when dApp Staking V3 was launched. As the total issuance changes (e.g. burn events), the dynamic thresholds will adjust accordingly, ensuring a fair and adaptive staking environment. However, the fixed threshold for the *lowest* tier remains constant.
```
Comment (Dinonard) [inline docs/learn/dapp-staking/dapp-staking-protocol.md:290]: Maybe it's a bit misleading to say the last part, that the fixed threshold remains a constant. It's still a percentage, that moves with total issuance.

File: docs/learn/dapp-staking/protocol-parameters.md [hunk 2]
```diff
@@ -65,11 +65,11 @@ E.g. `[5%, 15%, 20%, 60%]` means that `5%` of the total slots are assigned for *
...
 
-The threshold that dApp has to satisfy, essentially the total staked amount, in order to be eligible to enter a tier.
-There are two types of thresholds:
+The threshold a dApp must meet, based on a percentage of the total issuance staked, to enter a tier.
+There are two types of threshold percentages:
 
-* `Fixed` - threshold has a fixed value and doesn't change
-* `Dynamic` - threshold has an initial value, which changes dynamically as the number of slots change, but also has a minimum allowed value.
+* `Fixed` - A fixed percentage of the total issuance as staked funds, constant and unchanging between periods.
+* `Dynamic` - A percentage of the total issuance as staked funds that can change between periods, with a minimum percentage *minimum_required_percentage* that cannot be reduced below.
```
Comment (Dinonard) [inline docs/learn/dapp-staking/protocol-parameters.md:72]: Can it change between periods or eras now? (dumb question, just to be sure)

**Review by Dinonard — APPROVED** [review]

File: docs/learn/dapp-staking/dapp-staking-protocol.md (hunk 1 above)
Comment (ipapandinas) [reply docs/learn/dapp-staking/dapp-staking-protocol.md]: I agree, I just removed it.

File: docs/learn/dapp-staking/protocol-parameters.md (hunk 2 above)
Comment (ipapandinas) [reply docs/learn/dapp-staking/protocol-parameters.md]: Not the percentage itself, I've adjusted the explanation, thanks!

**Pull request merged**